_component_design/footer.md

```diff
@@ -12,6 +12,7 @@ last-modified: 2017-10-24
 helix-ui-css: na
 helix-ui-javascript: na
 helix-ui-note: Implemented by Pilot
```

Comment:
Did Pilot update to start handling the footer?

Reply:
hmmm, not that I know of.